module1/schedule.md

```diff
@@ -5,73 +5,68 @@ layout: page
 
 ## Week 1
 
-* **Technical Lessons**
-  * [Methods, Arguments, Return Values](./lessons/methods_and_return_values)
+* **Lessons**
   * [Objects, Instances and Classes](./lessons/objects_classes_instances)
   * [Intro to Testing](./lessons/intro_to_testing)
-  * [#each](./lessons/each)
   * [Flow Control](./lessons/flow_control)
   * [Scope](./lessons/scope)
-  * Intro to Debugging
-  * Learning to Pair
 
-* **Non-Technical Responsibilities**
+* **Other Responsibilities**
   * Professional Development
-  * Gear Up #1
-  * Accountabilibuddy Check-In
   * Community Building
-  * Cohort Retro
+  * House Time
+  * PD Discussion Groups
 
 * **Deliverables**
-  * Independent Challenge #1
   * Start [Solo Project](./projects)
 
 ## Week 2
 
-  * **Technical Lessons**
+  * **Lessons**
     * [Git and Github](./lessons/git_and_github)
-    * [Git for Pairs](./lessons/git_for_pairs)
-    * [Hashes](./lessons/introducing_hashes)
     * [Enumerables I](./lessons/enumerables_i)
+    * [Hashes](./lessons/introducing_hashes)
     * [Debugging with Pry](./lessons/debugging_with_pry)
     * [Debugging Techniques](./lessons/debugging_techniques)
-    * Reaching Through Objects
 
-  * **Non-Technical Responsibilities**
+  * **Other Responsibilities**
     * Professional Development
-    * Cohort Norms and Retro
+    * Cohort Norms
     * Accountabilibuddy Check-In
     * Giving and Receiving Feedback
     * One on One Instructor Meeting
+    * House Time
+    * Mix it Up Groups
 
   * **Deliverables**
-    * Submit [Solo Project](./projects)
-    * Independent Challenge #2
-    * Start [Pair Project](./projects)
+    * Submit [Solo Project #1](./projects)
+    * Week 2 Independent Challenge
+    * Start [Solo Project #2](./projects)
 
 ## Week 3
```

Comment:
I think we also have `Reaching Through Objects` in week 3

Reply:
Good catch!